docker-compose.yml

```diff
@@ -182,6 +192,23 @@ services:
       CONFIG: sshd_config
       PUB_KEY_NAME: id_rsa1.pub
 
+  http-proxy-chain1:
+    build: http-proxy
+    ports:
+      - '8889:3128'
```

Comment:
@thekondr Can we continue the port numbering under the current convention?

Reply:
@Maxbey what do you mean by the current convention? Do you mean port numbers starting from 2201? I was thinking that it would make sense to make the difference between SSH port numbers and HTTP proxy port numbers clear.